python/tank_vendor/shotgun_api3/lib/mockgun/mockgun.py

```diff
@@ -246,13 +246,6 @@ def schema_field_read(self, entity_type, field_name=None):
             return self._schema[entity_type]
         else:
             return dict((k, v) for k, v in self._schema[entity_type].items() if k == field_name)
-    
-    def get_next_id(self, entity_type):
```

Comment:
I understand why this was changed, but this broke the tests later so as you'll see.

Reply:
Stacey from Automotive had added this code directly to the Python API inside tk-core instead of doing it in the Python API repo, so it was lost when I updated the API. As you'll see later, I've reimplemented the logic in a similar fashion inside the tests. I've asked the automotive team and they didn't rely on this feature outside tk-core so we're good.